### PR TITLE
[initial] Add user controller and endpoint for getting user statistics

### PR DESCRIPTION
**Pull request by scratchw0w**

### CODE REVIEW

File: src/main/java/com/socio/rest/UserController.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package com.socio.rest;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/user")
+public class UserController {
```
Comment (oanzhi) [inline src/main/java/com/socio/rest/UserController.java:10]: why should we name this class as a "UserController"?

File: src/main/java/com/socio/rest/UserController.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+    @GetMapping
+    public ResponseEntity<String> getUsersStatistics() {
+        return ResponseEntity.ok("User statistics");
```
Comment (oanzhi) [inline src/main/java/com/socio/rest/UserController.java:14]: "User statistics", may we push this string to a const

File: src/main/java/com/socio/rest/UserController.java (hunk 1 above)
Comment (scratchw0w) [reply src/main/java/com/socio/rest/UserController.java:10]: I guess that this name seems the most proper for this situation

File: src/main/java/com/socio/rest/UserController.java (hunk 2 above)
Comment (scratchw0w) [reply src/main/java/com/socio/rest/UserController.java:14]: because this param will be removed in future

**Review by oanzhi — APPROVED** [review]
all seems fine to me

**Pull request merged**